apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -340,7 +351,7 @@ def render(element_html, data):
         mcq_options = data["params"][answer_name]
         mcq_options = filter_multiple_from_array(mcq_options, ["inner_html", "uuid"])
 
-        if answer_name in data["submitted_answers"]:
+        if answer_name in data["submitted_answers"]: 
```

Comment:
Make sure you run the black formatter over your code, you should be able to do this just with 

`make format-python`

Reply:
done